tests/requirements.txt

```diff
@@ -3,7 +3,7 @@ coverage==4.5.1
 Django==1.6.11  # Last version supporting 2.6.
 mock==2.0.0
 pytz==2018.5
-cffi==1.11.2  # Random pin to try and fix pyparser==2.18 not having effect
+cffi==1.14.3  # Random pin to try and fix pyparser==2.18 not having effect
```

Comment:
Have you upgraded cffi on general principal, or was it for something specific?

Reply:
I don't remember exactly, but I believe it's because of the new Ansible version. There was a reason, but I don't remember now 🤦 it's a newer version that works though haha, so improvement? 😅

Reply:
ok